transaction/summary/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name = "mc-transaction-summary"
+version = "4.0.2"
+authors = ["MobileCoin"]
+edition = "2021"
+readme = "README.md"
```

Comment:
it looks that we declared a README, but I don't see one in the diff? is that right? can we add one?